consensus/core/src/storage/rocksdb_store.rs

```diff
@@ -34,12 +31,14 @@ pub(crate) struct RocksDBStore {
     /// A secondary index that orders refs first by authors.
     digests_by_authorities: DBMap<(AuthorityIndex, Round, BlockDigest), ()>,
     /// Maps commit index to Commit.
-    commits: DBMap<(CommitIndex, CommitDigest), Bytes>,
+    commits: DBMap<CommitRef, Bytes>,
```

Comment:
Let's keep the explicit tuple as key. Someday we may need to ban constructing some values of CommitRef, but those values will still be needed as storage keys. Or we may want to add round to either CommitRef or storage key.